### DISCUSSION TITLE
Automatically sync/link current script with Project window scripts

### DISCUSSION THREAD

**Discussion by gsundberg-lyell** [dbeaver/dbeaver#21054] [Q&A] [answered]
There is an option in the Project to click "Link with editor", or shortcut ctrl-shift-, to highlight the script in the Project window for the currently active script being edited. It would nice to have an automatic option for this where the Project window automatically links/syncs with the scripts as the user simply clicks through open tabs.

The issue with the <ctrl-shift-,> shortcut is that it might link to the database navigator window and not the project window.

<img width="1557" alt="image" src="https://github.com/dbeaver/dbeaver/assets/106774808/1de265fb-0aaf-444c-9e6f-b4612f0f03b1">

**Comment by E1izabeth** [comment, marked as answer]
There is no such option. Here is an existing feature request #2054
